asf/sam/include/same70/component/xdmac.h

```diff
@@ -2396,8 +2460,8 @@ typedef struct {
   __O  uint32_t XDMAC_GSWR;     /**< (XDMAC Offset: 0x38) Global Channel Software Request Register */
   __I  uint32_t XDMAC_GSWS;     /**< (XDMAC Offset: 0x3C) Global Channel Software Request Status Register */
   __O  uint32_t XDMAC_GSWF;     /**< (XDMAC Offset: 0x40) Global Channel Software Flush Request Register */
-  RoReg8  Reserved1[0x0C];
-       XdmacChid XDMAC_CHID[XDMACCHID_NUMBER]; /**< Offset: 0x50 Channel Interrupt Enable Register (chid = 0) */
+  __I  uint8_t                        Reserved1[12];
+       XdmacChid XdmacChid[XDMACCHID_NUMBER]; /**< Offset: 0x50 Channel Interrupt Enable Register (chid = 0) */
```

Comment:
In the old implementation this field was called `XDMAC_CHID`, now we have `XdmacChid`. This is causing a compile time error in `drivers/dma/dma_sam_xdmac.c`.

Reply:
Thanks for reviewing.

> In the old implementation this field was called `XDMAC_CHID`, now we have `XdmacChid`. This is causing a compile time error in `drivers/dma/dma_sam_xdmac.c`.

Ok, I will add a patch to fix that name.

Reply:
I didn't go through all the files, there may be more places that need fixing.